api/tests/routes/native/v1/banner_test.py

```diff
@@ -190,7 +190,7 @@ def should_get_17_18_transition_id_check_todo_banner(self, client):
 
     def should_get_17_18_transition_id_check_done_banner(self, client):
         a_year_ago = datetime.datetime.utcnow() - relativedelta(years=1, months=1)
```

Comment:
(détail) on peut sans doute écrire ça  plus proprement
```suggestion
        with time_machine.travel(datetime.datetime.utcnow() - relativedelta(years=1, months=1))
```

Reply:
Tu trouves plus propre de ne pas mettre le delta dans une variable ?